app/pom.xml

```diff
@@ -80,6 +80,12 @@
 			<artifactId>xmlsec</artifactId>
 			<version>1.3.0</version>
 		</dependency>
+		<dependency>
+			<groupId>wsdl4j</groupId>
+			<artifactId>wsdl4j</artifactId>
+			<version>1.6.3</version>
```

Comment:
The version should be pulled up into the root `pom.xml` file.  You'll see the pattern for this in that file.